### ISSUE TITLE
.babelrc file causing error

### ISSUE THREAD

**Issue by pantchox** [threepointone/react-ease#6]
when building with webpack after npm installtion it says the .babelrc file is in old syntax.
please update it :+1:

**Comment by threepointone** [comment]
happy to take a PR if it's urgent, else I'll get to this on the weekend.

**Comment by pantchox** [comment]
No Rush bro, thing is that the new babelrc format for the defiend '0' stage, requires to install a babel npm module, and i don't know if that is what you think is appropriate  ;)

**Comment by pantchox** [comment]
do we actaully need this? can i remove the .babelrc file? - sorry i am new to NPM and all that stuff

**Comment by threepointone** [comment]
yes it's needed for the build. annoyed that babel reaches into node_modules to grab this, but I know a workaround. gimme a few hours, and I'll fix this tonight, k?

**Comment by pantchox** [comment]
thanks that will be great :D
